networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/DataColumnSidecarsByRangeMessageHandler.java

```diff
@@ -58,6 +58,7 @@ public class DataColumnSidecarsByRangeMessageHandler
         DataColumnSidecarsByRangeRequestMessage, DataColumnSidecar> {
 
   private static final Logger LOG = LogManager.getLogger();
+  private static final Logger LOG_DAS = LogManager.getLogger("das-nyota");
```

Comment:
We have found that at least in Kurtosis this info is no collected, do we really need separate loggers for this?